packages/oas-normalize/README.md

```diff
@@ -27,126 +27,161 @@ npm install oas-normalize
 
 ## Usage
 
-```javascript
+```ts
 import OASNormalize from 'oas-normalize';
-// const { default: OASNormalize } = require('oas-normalize'); // If you're using CJS.
 
 const oas = new OASNormalize(
   'https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore-expanded.yaml',
-  // ...or a string, path, JSON blob, whatever you've got.
+  // ...or a JSON object, YAML, a file path, stringified JSON, whatever you have.
 );
 
-oas
+await oas
   .validate()
-  .then(definition => {
-    // Definition will always be JSON, and valid.
-    console.log(definition);
+  .then(() => {
+    // The API definition is valid!
   })
   .catch(err => {
-    console.log(err);
+    console.error(err);
   });
 ```
 
-### `#bundle()`
+### `.load()`
```

Comment:
`.load()` has long been marked as private with a `@private` JSDoc but we actually use it in a number of apps of ours so might as well make it _actually_ public and documented.